src/OnlineSales/appsettings.json

```diff
@@ -70,6 +70,13 @@
       "RetryCount": "2",
       "RetryInterval": "5",
       "BatchSize": "5000"
+    },
+    "EmailLogsToActivityLogDumpTask": {
```

Comment:
@Exmirai rename to SyncEmailLogTask to make the name consistent with SyncSmsLogTask.